common/api/ecschema-metadata.api.md

```diff
@@ -488,6 +490,14 @@ export class ECVersion {
     get write(): number;
 }
 
+// @internal
+export interface ECXmlVersion {
+    // (undocumented)
+    readVersion: number;
+    // (undocumented)
+    writeVersion: number;
+}
+
```

Comment:
I want to move away from calling this the ECXml version because it applies to the JSON as well ... we even do it in this code where we set the xml version from the json version.  I propose ECSpec instead of ECXml.

Reply:
Cool, I'll create a backlog item on native to track it.